src/components/Composer.vue

```diff
@@ -462,6 +461,21 @@ export default {
 		allRecipients() {
 			this.checkRecipientsKeys()
 		},
+		aliases(newAliases) {
+			console.debug('aliases changed')
+			if (this.selectedAlias === NO_ALIAS_SET) {
+				return
+			}
+
+			const newAlias = newAliases.find(alias => alias.id === this.selectedAlias.id && alias.aliasId === this.selectedAlias.aliasId)
```

Comment:
:thinking: :thinking: :thinking: 

so ``alias.id`` is the *account id* and ``alias.aliasId`` is the *alias id*?

Reply:
![image](https://user-images.githubusercontent.com/3902676/114699403-8b820380-9d20-11eb-8a0b-4f4720f971be.png)

Yes

Reply:
could we flip that so that ``alias.id`` is the *alias id?

and for the comparison it should be sufficient to compare the alias IDs as those are unique, right?